### ISSUE TITLE
protocols.http.request method documentation error

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8031]
|<img alt="bkc's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| bkc reported|
|-|-|
|Trac ID|trac#453|
|Type|defect|
|Created|2003-12-29 23:10:31Z|

<details><summary>Searchable metadata</summary>

```
trac-id__453 453
type__defect defect
reporter__bkc bkc
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1072739431000000 1072739431000000
changetime__1072739431000000 1072739431000000
version__ 
owner__ 
cc__bkc
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="bkc's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|bkc commented|
|-|-|

```
#!html
<pre>
In the documentation included with the twisted 1.1.1 .tar file, 
at /api/twisted.protocols.http.Request.html,

method is listed as a class variable, but I think it should be an instance 
variable.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
